src/main/kotlin/calculator/StringInputSplitter.kt

```diff
@@ -2,32 +2,39 @@ package calculator
 
 import java.math.BigDecimal
 
-object StringInput {
+object StringInputSplitter {
     private const val START_LINE = "//"
-    private const val DEFAULT_REGEX = "[,:]"
-    private const val CUSTOM_REGEX = "//(.)\\n(.*)"
+    private const val CUSTOM_REGEX = "//(.)\n(.*)"
     private const val CUSTOM_SEPARATOR_INDEX = 1
     private const val CUSTOM_TOKENS_INDEX = 2
 
-    fun stringToBigDecimalList(stringNumbers: String?): List<BigDecimal>? =
-        if (stringNumbers.isNullOrBlank()) {
-            listOf(BigDecimal.ZERO)
-        } else if (stringNumbers.contains(START_LINE)) {
-            splitToCustomText(stringNumbers)
-        } else {
-            splitToDefaultText(stringNumbers)
+    private val defaultRegex = "[,:]".toRegex()
+
+    fun stringToBigDecimalList(stringNumbers: String?): List<BigDecimal> =
+        when {
+            stringNumbers.isNullOrBlank() -> {
+                listOf(BigDecimal.ZERO)
+            }
+            stringNumbers.contains(START_LINE) -> {
+                splitToCustomText(stringNumbers)
+            }
+            else -> {
+                splitToDefaultText(stringNumbers)
+            }
```

Comment:
코틀린 when절에 case 당 1줄 밖에 없다면, 중괄호 없이 사용하는 것이 컨벤션입니다 :)
```suggestion
        when {
            stringNumbers.isNullOrBlank() -> listOf(BigDecimal.ZERO)
            stringNumbers.contains(START_LINE) -> splitToCustomText(stringNumbers)
            else -> splitToDefaultText(stringNumbers)
        }
```